app/views/admin/topics/_ticket.html.erb

```diff
@@ -6,7 +6,7 @@
 
   <div id="ticket-controls" class="col-md-8 col-xs-12 col-lg-8">
     <h2>
-      <%= render 'admin/topics/ticket_nav_dropdown' %> #<%= @topic.id %>- <%= @topic.name -%>
+      <%= render 'admin/topics/ticket_nav_dropdown' %> #<%= @topic.id %>- <%= best_in_place @topic, :name, as: :input, url: admin_topic_path(@topic), activator: :span -%>&nbsp;<span class="btn" id="btn-edit-topic-name">Edit</span>
```

Comment:
Can you make this `class="btn btn-default"` it should match the other buttons and you won't need to style it to 10px.